storage/elasticsearch/client.go

```diff
@@ -47,24 +47,42 @@ const indexVersion = 11
 const indexPrefix = "skydive"
 const indexAllAlias = "all"
 
-// ElasticLimits describes index limits driving roll policy
-type ElasticLimits struct {
+// Config describes configuration for elasticsearch
+type Config struct {
+	ElasticHost  string
+	MaxConns     int
+	RetrySeconds int
+	BulkMaxDocs  int
+	BulkMaxDelay int
 	EntriesLimit int
 	AgeLimit     int
 	IndicesLimit int
 }
 
-// NewElasticLimitsFromConfig create new limits from configuration
-func NewElasticLimitsFromConfig(path string) ElasticLimits {
-	limits := ElasticLimits{}
-	limits.EntriesLimit = config.GetInt(path + ".index_entries_limit")
-	limits.AgeLimit = 0
+func NewConfig(name string) Config {
```

Comment:
NewConfig(name ...string)
if len(name) > 0 ...

to avoid call like NewConfig("")

Reply:
ok